app/main.py

```
@@ -4,6 +4,7 @@
 from fastapi import FastAPI, Request, status
 from fastapi.responses import JSONResponse
 from fastapi.exceptions import RequestValidationError
+from app.schemas import *
```

Ruff (0.14.11)

`from app.schemas import *` used; unable to detect undefined names

(F403)